etc/environment-postprocess.yml

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
The reason I changed it is that it is now according to [specification](https://yaml.org/spec/1.2.2/#22-structures) and the linter found it.

It is often done otherwise and it seems odd.

As there is only a single document per file we could also leave them away. That is not strictly compliant with spec, but there is no reason for an error, as there is only a single document per file. This is way more important if you use a stream instead of a file.

Reply:
Shall I remove them again to improve readability (to many people)? Or shall we leave as is now?

Reply:
No, better leave it if this is the standard somehow.